zerver/models/groups.py

```diff
@@ -94,6 +95,16 @@ class NamedUserGroup(UserGroup):  # type: ignore[django-manager-missing] # djang
     }
 
     GROUP_PERMISSION_SETTINGS = {
+        "can_join_group": GroupPermissionSetting(
+            require_system_group=False,
+            allow_internet_group=False,
+            allow_owners_group=True,
+            allow_nobody_group=False,
+            allow_everyone_group=False,
```

Comment:
I think a default of nobody might be more reasonable for this group; we can use a non-nobody group for "who can add anyone to a group" to allow owners to subscribe themselves.

Reply:
Updated the default.